test/testsprite.c

```diff
@@ -495,6 +499,10 @@ SDL_AppResult SDL_AppInit(void **appstate, int argc, char *argv[])
         return SDL_APP_FAILURE;
     }
 
+    for (i = 0; i < state->num_windows; i++) {
```

Comment:
Does this override command line parameters? Debug code?

Reply:
Debug code, removed in f7093e6ec1f601ced1cf8f6f624c40baabc6e274.